### PR TITLE
Include Chrome DevTools color contrast feature

### LINKED ISSUE #4315: content: Provide information about the future contrast algorithm APCA
### What page(s) need to be updated?

https://web.dev/color-and-contrast-accessibility/

### Why is this update needed?

Chrome DevTools has implemented the support for APCA as an experiment. DevTools links to https://web.dev/color-and-contrast-accessibility/ to provide more context about contrast threshold such as AA/AAA. It'd be helpful to include some information about the new algorithm called APCA that is intended as a replacement for AA/AAA in the future. More context:

1. crbug (https://bugs.chromium.org/p/chromium/issues/detail?id=1121900)
2. Visual Contrast explainer (https://w3c.github.io/silver/guidelines/explainers/visualContrast.html)
3. APCA calculator (https://www.myndex.com/APCA/)

Note that APCA is still an experiment and it is subject to changes.

### What's the deadline?

no deadline

#### Is it a hard deadline?

no

### PR DESCRIPTION
**Pull request by chybie**
Fixes #4315

### CODE REVIEW

**Comment by chybie** [comment]
PTAL @OrKoN

File: src/site/content/en/accessible/color-and-contrast-accessibility/index.md [hunk 1]
```diff
@@ -54,6 +60,24 @@ You will get a detailed report of any failing elements.
   <img src="./fastpass-contrast.jpg" alt="The report in Accessibility Insights">
 </figure>
 
+## Advanced Perceptual Contrast Algorithm (APCA)
+
+The [Advanced Perceptual Contrast Algorithm (APCA)](https://w3c.github.io/silver/guidelines/methods/Method-font-characteristic-contrast.html) is a new way to compute contrast based on modern research on color perception.
```
Comment (chybie) [inline src/site/content/en/accessible/color-and-contrast-accessibility/index.md:65]: I did not explain about the Silver Task Force here: https://w3c.github.io/silver/. Let me know if you think it's good to do so. @mathiasbynens

**Review by mathiasbynens — APPROVED** [review]

**Comment by mathiasbynens** [comment]
LGTM, thanks! As an aside: we should probably add some documentation (not here, but rather to the DevTools docs or What's New blog post) on how to interpret the APCA results, and how to read the percentages.

File: src/site/content/en/accessible/color-and-contrast-accessibility/index.md [hunk 2]
```diff
@@ -54,6 +60,24 @@ You will get a detailed report of any failing elements.
...
+
+Compared to [AA](https://www.w3.org/WAI/WCAG21/quickref/#contrast-minimum)/[AAA](https://www.w3.org/WAI/WCAG21/quickref/#contrast-enhanced) guidelines, APCA is more context-dependent.
+
+The contrast is calculated based on the text's:
+
+- spatial properties (font weight & size),
+- color (perceived lightness difference between text and background),
+- context (ambient light, surroundings, intended purpose of the text).
```
Comment (mihajlija) [inline src/site/content/en/accessible/color-and-contrast-accessibility/index.md:73]:
```suggestion
The contrast is calculated based on the following features:

- text's spatial properties (font weight and size)
- text color (perceived lightness difference between text and background)
- context (ambient light, surroundings, and intended purpose of the text)
```

**Review by mihajlija — CHANGES_REQUESTED** [review]
Thanks for updating the post! Please optimize the images and I'll merge it promptly :)

Here's a helpful guide for that: https://web.dev/handbook/markup-media/#optimization

File: src/site/content/en/accessible/color-and-contrast-accessibility/index.md [hunk 3]
```diff
@@ -54,6 +60,24 @@ You will get a detailed report of any failing elements.
...
+
+The [Advanced Perceptual Contrast Algorithm (APCA)](https://w3c.github.io/silver/guidelines/methods/Method-font-characteristic-contrast.html) is a new way to compute contrast based on modern research on color perception.
+
+Compared to [AA](https://www.w3.org/WAI/WCAG21/quickref/#contrast-minimum)/[AAA](https://www.w3.org/WAI/WCAG21/quickref/#contrast-enhanced) guidelines, APCA is more context-dependent.
+
+The contrast is calculated based on the following features:
+
+- text's spatial properties (font weight and size)
+- text color (perceived lightness difference between text and background)
+- context (ambient light, surroundings, and intended purpose of the text)
```
Comment (mihajlija) [inline src/site/content/en/accessible/color-and-contrast-accessibility/index.md:73]:
```suggestion
- Spatial properties (font weight and text size)
- Text color (perceived lightness difference between text and background)
- Context (ambient light, surroundings, and intended purpose of the text)
```

**Comment by chybie** [comment]
Thanks @mihajlija ! Optimised images.

**Comment by chybie** [comment]
Noted. @mathiasbynens

**Review by mihajlija — APPROVED** [review]

**Pull request merged**